app/assets/javascripts/activeadmin_addons/inputs/search-select.js

```diff
@@ -0,0 +1,71 @@
+$(function() {
```

Comment:
Definition for rule 'import/export' was not found                 import/export<br>Definition for rule 'import/no-mutable-exports' was not found     import/no-mutable-exports<br>Definition for rule 'import/no-amd' was not found                 import/no-amd<br>Definition for rule 'import/imports-first' was not found          import/imports-first<br>Definition for rule 'import/no-duplicates' was not found          import/no-duplicates<br>Definition for rule 'import/newline-after-import' was not found   import/newline-after-import<br>Definition for rule 'import/prefer-default-export' was not found  import/prefer-default-export